pkg/fanal/analyzer/language/nodejs/yarn/yarn_test.go

```diff
@@ -230,6 +230,103 @@ func Test_yarnLibraryAnalyzer_Analyze(t *testing.T) {
 			dir:  "testdata/unsupported_protocol",
 			want: &analyzer.AnalysisResult{},
 		},
+		{
+			name: "monorepo",
+			dir:  "testdata/monorepo",
+			want: &analyzer.AnalysisResult{
+				Applications: []types.Application{
```

Comment:
Correct me if I'm wrong, but it looks like all dependencies for this test should be `Indirect`.

Reply:
Workspaces are part of the project and we consider it as one application, so can we consider their direct dependencies as indirect?

Reply:
I think it would make sense if we considered all projects as separate applications, wdyt?

Reply:
hm... it is strange for me, but looks like you are right:
```
npm list --depth=0
app@1.0.0 /app
`-- package1@1.0.0 -> ./package1 ## it is workspace
  `-- debug@4.3.4
```